web/themes/custom/cloudy/templates/content/location/node--location--teaser.html.twig

```diff
@@ -6,30 +6,31 @@
     node.isSticky() ? 'node--sticky',
     not node.isPublished() ? 'node--unpublished',
     view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
-    'mb-3',
+    'mb-4',
```

Comment:
Added location templates to all use similar classes. Long term, I would love to drop the mb classes in favor of applying space to the display modes themselves (teaser, search result, related, etc.).